crates/bevy_ecs/src/system/system.rs

```diff
@@ -41,6 +41,12 @@ pub trait System: Send + Sync + 'static {
     /// Returns true if the system must be run exclusively.
     fn is_exclusive(&self) -> bool;
 
+    /// calling this prevents schedule from adding a auto sync point after this system
+    fn no_sync_after(&mut self);
```

Comment:
I feel like it would make more sense for this to be a method of `SystemConfig`, not `System`

Reply:
I added it on system, because that is what knows if it has deferred system params or not.

Reply:
thought about this some more, and I'll try adding some data on `GraphInfo` for this.